documentation/asciidoc/topics/proc_restoring_cluster.adoc

```diff
@@ -0,0 +1,51 @@
+[id='restoring_clusters-{context}']
+= Restoring {brandname} Clusters
+Restore {brandname} cluster state from backup archives.
+When you restore cluster state, {ispn_operator} applies all caches and data from the backup archive on to the target cluster.
```

Comment:
It's also possible to selectively restore content from a backup via:

```yaml
spec:
  resources:
    caches:
      - some-cache
      - and-another-one
    counters:
      - counter-name
```

Reply:
@ryanemerson these are all the available options under `spec.resources` right?

Reply:
Bingo!

Reply:
thx. am kicking myself a little for not looking there in the first place. one of my questions, that I didn't ask, was about selective restore. I think those resources are pretty self-explanatory so I didn't add callouts for each one. hopefully the examples make them sufficiently clear too.